refs/biblio.json

```diff
@@ -2425,6 +2425,13 @@
     "SMPTE2052-1": {
         "aliasOf": "SMPTE2052-1-2013"
     },
+    "SMPTE296": {
+        "title": "ST 296:2012 - SMPTE Standard - 1280 × 720 Progressive Image 4:2:2 and 4:4:4 Sample Structure — Analog and Digital Representation and Analog Interface",
```

Comment:
I would set `title` to `ST 296:2012, 1280 × 720 Progressive Image 4:2:2 and 4:4:4 Sample Structure — Analog and Digital Representation and Analog Interface`

Reply:
done